dashboard/app/models/regional_partner.rb

```diff
@@ -139,6 +139,7 @@ def upcoming_summer_workshops
   validates_inclusion_of :applications_decision_emails, in: APPLICATION_DECISION_EMAILS, if: -> {applications_decision_emails.present?}
   validates :csd_cost, numericality: {greater_than: 0}, if: -> {csd_cost.present?}
   validates :csp_cost, numericality: {greater_than: 0}, if: -> {csp_cost.present?}
+  validates :is_active, inclusion: [true, false]
```

Comment:
this seems a bit redundant for a boolean field. are you just trying to enforce that the field is not `nil`? if so, you could do it with `validates_presence_of :is_active`

Reply:
Yes, thanks for the suggestion, this is much better to read.